docs/scripts/api/buildComponentsDocumentation.ts

```diff
@@ -53,7 +54,7 @@ export interface ReactApi extends ReactDocgenApi {
   src: string;
   styles: Styles;
   displayName: string;
-  slots: Record<string, { default: string | undefined; type: { name: string | undefined } }>;
+  slots?: Slot[];
```

Comment:
Reuse the ReactApi interface to avoid mismatches like this?

```ts
export interface ReactApi extends Omit<CoreReactApi, 'propsTable' | 'translations' | 'classes'> {
  displayName: string;
  packages: { packageName: string; componentName: string }[];
}
```